Dockerfile

```diff
@@ -1,35 +1,51 @@
+# ---- Base Node ----
 FROM node:18.13.0-bullseye-slim as base
 
-RUN set -ex; \
-    apt update -y; \
-    apt install ca-certificates -y; \
-    update-ca-certificates
-
-FROM base as build
-
-WORKDIR /app
-
-RUN apt install make gcc g++ python3 git openssl -y;
+RUN mkdir -p /home/node/app
+RUN chown -R node:node /home/node && chmod -R 770 /home/node
+WORKDIR /home/node/app
+
+# ---- Dependencies ----
+FROM base AS dependencies
+
+COPY --chown=node:node ./package.json ./package.json prisma ./
+COPY --chown=node:node ./package-lock.json ./package-lock.json
+RUN apt-get update -y && \
+    apt-get install -y --no-install-recommends \
+      ca-certificates \
+      dumb-init \
+      make \
+      gcc \
+      g++ \
+      python3 \
+      git \
+      openssl
+USER node
+# install production dependencies
+RUN npm ci
```

Comment:
```suggestion
RUN npm ci  --ignore-scripts;
```

Reply:
This breaks `@prisma/client`. I will look into it further how to explicitly bootstrap it while using `--ignore-scripts`.

Reply:
"npx prisma generate" does it